### DISCUSSION TITLE
Missing ENV

### DISCUSSION THREAD

**Discussion by mvyleteljr** [adonisjs/core#3214] [Help] [answered]
I'm trying to run some new db migrations to alter schema and every time I run:

node ace make:migrations {migrationName} I get the following error:

`Exception: E_MISSING_ENV_VALUE: Missing environment variable "HOST"`

I have .env.api files in /build/development with a defined HOST value, and I have tried moving the files to the project root, inside the api/build/ files, and everywhere else i could think to put it, but nothing is working. This is an incredibly frustrating error. Any thoughts?

**Comment by mvyleteljr** [comment]
UPDATE: When running node ace serve --watch just to see what happens, I also get 

`Exception: E_MISSING_ENV_VALUE: Missing environment variable "APP_KEY"`

Even though it is, again, defined in my .env files and i've put the files in what I believe is the right spot

**Comment by thetutlage** [comment, marked as answer]
> I have .env.api files in /build/development with a defined HOST value, and I have tried moving the files to the project root

I have no idea what is `.env.api` and what is `build/development`. Documentation doesn't talk about any of these

**Reply by mvyleteljr** [reply]
it was a docker issue i fixed, my bad
